backend/src/main/java/gov/cdc/usds/simplereport/db/model/BaseTestInfo.java

```diff
@@ -19,19 +19,19 @@
   @JoinColumn(name = "patient_id", updatable = false)
   private Person patient;
 
-  @ManyToOne(optional = false)
+  @ManyToOne(optional = false, fetch = FetchType.LAZY)
   @JoinColumn(name = "organization_id", updatable = false)
   private Organization organization;
 
-  @ManyToOne(optional = false)
+  @ManyToOne(optional = false, fetch = FetchType.LAZY)
   @JoinColumn(name = "facility_id", updatable = false)
   private Facility facility;
 
-  @ManyToOne(optional = false)
+  @ManyToOne(optional = false, fetch = FetchType.LAZY)
```

Comment:
This will potentially blow up the export query, don't know if you've already dealt with that. Though I guess in the worst case we just ... fetch it with an N+1 query plan.

Reply:
Removing `DeviceSpecimenType deviceSpecimen` Lazy fetch